en/xml/fo/topic.graphic-pageheader.xml

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+    License CC BY-NC-SA 3.0
+
+   This work is licensed under the
+   "Namensnennung – Keine kommerzielle Nutzung – Weitergabe unter
+    gleichen Bedingungen 3.0 Deutschland (CC BY-NC-SA 3.0)"
+   http://creativecommons.org/licenses/by-nc-sa/3.0/de/deed.de
+
+   Read the English translation here:
+
+   "Attribution-NonCommercial-ShareAlike 3.0 Unported (CC BY-NC-SA 3.0)"
+   http://creativecommons.org/licenses/by-nc-sa/3.0/
+
+-->
+<?xml-model href="file:../5.1/dbref.rnc" type="application/relax-ng-compact-syntax"?>
+<section xml:id="dbc.fo.graphic-pageheader" remap="topic" userlevel="easy"
+   xmlns="http://docbook.org/ns/docbook"
+   xmlns:xi="http://www.w3.org/2001/XInclude"
+   xmlns:xlink="http://www.w3.org/1999/xlink">
+   <title>Inserting Graphics to your Page Headers or Footers</title>
+   <info>
+       <author>
+           <personname>
+               <firstname>Peter</firstname>
+               <surname>Schmelzer</surname>
+           </personname>
+       </author>
+       <keywordset>
+         <keyword>page header</keyword>
+         <keyword>block</keyword>
+         <keyword>format</keyword>
+         <keyword>page</keyword>
+      </keywordset>
+      <subjectset>
+         <subject>
+            <subjectterm>graphic</subjectterm>
+         </subject>
+      </subjectset>
+   </info>
+   
+   <section role="problem">
+      <title>Problem</title>
+      <para>You need to know how to insert a graphic in your page header or footer.</para>
+   </section>
+
+   <section role="solution">
+      <title>Solution</title>
+      <para>First of all, you need to create a customization layer in your 
+         customization stylesheet containing the following code:</para>
+      <programlisting>&lt;!-- Path and name of your graphic file-->
```

Comment:
Further ideas which makes handling of such files easier:

* Use `<example>` around your `programlisting` element.
   This makes it easier to refer to it and you can give it a title and name (see [Creating Permalinks, Example 5.1](http://doccookbook.sourceforge.net/html/en/dbc.html.permalinks.html)
* Outsource your XSLT stylesheet into a separate file and `xi:include` it in your XML source.
  This helps in making example files and (more important!) gives our readers the chance to download this file
* Reduces the size of your XML file.

If you like this approach, I'd recommend the following:

1. Create a directory `logo` under `fo`.
1. Save you XSLT stylesheet in `fo/logo/YOUR_STYLESHEET.xsl`. Replace the placeholder `YOUR_STYLESHEET.xsl` with some useful name.
1. Replace the content of your stylesheet code in your XML file with the following content:

```xml
<example xml:id="ex.fo.XXXXX">
   <title>Some good title (<filename>YOUR_STYLESHEET.xsl</filename>)</title>
   <info>
        <output xmlns="urn:x-toms:docbook-ext">
          <filename>logo/YOUR_STYLESHEET.xsl</filename>
        </output>
   </info>
   <programlistingco>
      <areaspec>
            <!-- Add your callouts through <area/> -->
      </areaspec>
      <programlisting language="xml" linenumbering="numbered"
          ><xi:include href="logo/YOUR_STYLESHEET.xsl" parse="text"
          /></programlisting>
   </programlistingco>
</example>
```

Reply:
Done, but with a similar way. I've placed all necessary files below the folder "graphic-pageheader/".